### ISSUE TITLE
Placeholder text color

### ISSUE THREAD

**Issue by sausagemountain** [Wox-launcher/Wox#2833]
### Are you submitting a bug report?

No

While making a theme there is no option to customize the color of placeholder text, it's always the same colour which really harms visibility even on BlurWhite theme

![placeholder](https://user-images.githubusercontent.com/22707290/79867884-7833a500-83f8-11ea-808f-95ea721dfedb.png)
![suggestion](https://user-images.githubusercontent.com/22707290/79867892-79fd6880-83f8-11ea-9aae-6d947e9a5fe4.png)

**Comment by bao-qian** [comment]
it can be adjusted in theme file, 
make a copy of the BlurWhite.xml theme file you want, 
place it inside %APPDATA%\Wox\Themes, and give it another name.
change the Foreground color value for different style key, restart wox and choose the new theme to take effect.
or you can make a pr to submit a theme file with better font color.

**Comment by sausagemountain** [comment]
What is the style name for this text? I do not think I can change it in a theme, as I only see that in MainWindow.xaml the style for it is QueryBoxStyle and I am changing it and nothing happens. Also I cannot style an element based on its name, which is what I would like to do.
![red text](https://user-images.githubusercontent.com/22707290/79979630-3fa8cf80-84bb-11ea-88e0-2b0618ff887a.PNG)

**Comment by bao-qian** [comment]
sorry, could you rephrase your question? I really don't understand what you mean by "style name" and "style".

**Comment by sausagemountain** [comment]
I kind of fixed it [in this pull request](https://github.com/Wox-launcher/Wox/pull/2839)

**Comment by bao-qian** [comment]
great
